src/v/kafka/server/handlers/configs/config_utils.h

```diff
@@ -519,6 +520,25 @@ struct iceberg_partition_spec_validator {
     }
 };
 
+struct iceberg_target_lag_ms_validator {
+    std::optional<ss::sstring> operator()(
+      const ss::sstring& /*raw*/,
+      const std::optional<std::chrono::milliseconds>& maybe_value) {
+        if (maybe_value.has_value()) {
+            const auto& value = maybe_value.value();
+            constexpr auto min_lag = std::chrono::milliseconds(10s);
+            if (value < min_lag || value > serde::max_serializable_ms) {
+                return fmt::format(
+                  "target.lag.ms value invalid, expected to be in range "
```

Comment:
do you need to specify field name here? i remember seeing it automatically sent back to the client in some places

Reply:
ugh, good shout. not entirely sure. iirc this is basically copy/paste from the one for `delete.retention.ms`

Reply:
several of the other messages include the name, so I'm going to leave it. 

can poke at them async and do a bulk change if it turns out to be redundant.